### DISCUSSION TITLE
语音 Based 的 AI 待办事项

### DISCUSSION THREAD

**Discussion by lapse12** [creatorsand/.github#83] [Ideas]
一句话 IDEA：
通过语音输入生成待办事项，自动分类和设置优先级，融入四象限等任务管理理念

我的介绍&角色：
KleinHE，全干开发者 👾，业余 UI/UX 🪄，更多请看我个人网站：kleinhe.com
这次项目中主要负责开发

期待的共创：
这次主要需要做设计和 mkt/运营 的朋友一起共创！（开发应该这次共学就能完整做出来）
也欢迎开发的朋友找我，之后应该也有很多可以一起合作的机会～

为什么想做这个 App ：
1. 因为我自己就有 ToDo 管理的需求，但现在 Apple 原生的待办事项不支持语音输入
2. 这个品类从来不缺竞品，我自己也体验过很多，但绝大多数都无法同时做到 核心功能的完备&良好的 UX（UI + 体验流程）
所以我就想快速 build 一个真正切实、优雅地解决问题的应用

这件事的难点：
1. 把用户随意（偶有语病和重复）的语音输入转为结构化良好的待办事项输出
2. 极简且优雅的设计

产品理念：
1. 轻量，追求极快的打开速度
2. 极简设计，一切 UI 以功能为核心，拒绝花里胡哨

**Comment by Cobb9527** [comment]
个人网站做的不错哈哈哈~
